File: telegram/src/main/java/by/mrrockka/TelegramApplication.java
package by.mrrockka;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.context.annotation.ComponentScan;
import org.springframework.context.annotation.EnableAspectJAutoProxy;

@SpringBootApplication
@EnableAspectJAutoProxy(proxyTargetClass = true)
@ComponentScan({"by.mrrockka", "org.telegram.telegrambots"})
public class TelegramApplication {

  public static void main(final String[] args) {
    SpringApplication.run(TelegramApplication.class, args);
  }
}

File: telegram/src/main/java/by/mrrockka/TelegramApplication.kt
package by.mrrockka

import org.jetbrains.exposed.v1.spring.boot4.autoconfigure.ExposedAutoConfiguration
import org.springframework.boot.autoconfigure.ImportAutoConfiguration
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.context.properties.EnableConfigurationProperties
import org.springframework.boot.runApplication
import org.springframework.scheduling.annotation.EnableScheduling

@EnableScheduling
@EnableConfigurationProperties
@ImportAutoConfiguration(value = [ExposedAutoConfiguration::class])
@SpringBootApplication
open class TelegramApplication

fun main(args: Array<String>) {
    runApplication<TelegramApplication>(*args)
}

